software/lyrion-music-server.yml

```
@@ -0,0 +1,14 @@
+name: "Lyrion Music Server"
+website_url: "https://lyrion.org/"
+source_code_url: "https://github.com/lms-community/slimserver"
+description: "Server for Squeezebox and compatible players."
+licenses:
+  - GPL-2.0
+platforms:
+  - deb
+  - Docker
+  - Shell 
```

```suggestion
  - Perl
```
Since it also distributes binaries, we should use the Source language